.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,92 @@
+          - If you want to report a bug for the Phoscon App, please head over to: https://forum.phoscon.de
+          - Make sure not to remove any headers and fill the template completely. If you remove the headers, the issue will be auto-closed.
+          - If you're unsure if the bug fits into this issue tracker, please ask for advise in our Discord chat: https://discord.gg/QFhTxqN
+          - Please make sure sure you're running the latest version of deCONZ: https://github.com/dresden-elektronik/deconz-rest-plugin/releases
+  - type: checkboxes
+    attributes:
+      label: I truly want to report a bug
+      description: |
+        I've read the important notes, am aware of [#5113](https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5113) and confirm I'm not issuing a question.
+      options:
+      - label: I definitively want to report a bug
+        required: true
+  - type: checkboxes
+    attributes:
+      label: Is there already an existing issue for this?
+      description: Please search this repository to see if an issue already exists for the bug you encountered.
+      options:
+      - label: I have searched the existing issues
+        required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: |
+        Describe the issue you are experiencing here to communicate to the maintainers. Tell us what you were trying to do and what happened. Help us understand the issue by providing valuable context.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce the behavior
+      description: |
+        If the problem is reproducable, list the steps here. If the problem can't be reproduced and is sporadic, please provide some details on how often and when the issue happens.
+      placeholder: |
+          1. Go to '...'
+          2. Click on '....'
+          3. Scroll down to '....'
+          4. Observed error
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: If applicable, describe what you expected to happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain the problem.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        The deCONZ and firmware versions are found in:
+        - Phoscon App: Menu -> Settings -> Gateway, OR
+        - deCONZ GUI: Menu > Help > About deCONZ.
+      value: |
+           - Host system: (Raspberry Pi / PC / NAS / MAC)
+           - Running method: (Raspbian / Ubuntu / Home Assistent deCONZ Add-on / Marthoc Docker container / Windows / Virtual Machine)
+           - Firmware version: (26xxyy00)
+           - deCONZ version: (2.xx.yy)
```

Comment:
Perhaps change this to : deCONZ version (not Home assistant Addon version!):

Reply:
Then it is probably better to throw it out completely (Home Assistent deCONZ Add-on I mean)?

Reply:
Not really. People tend to put in the addon version which is a different number then the deconz version. The addon uses different versioning numbers.

Reply:
Ah, now I got what you mean :) Done